### PR TITLE
Added F-Droid download link

### PR DESCRIPTION
**Pull request by qwow5**
Added a download link for the F-Droid app on the Android Flavors page under the minimal section.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @qwow5, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;companion.home-assistant#405).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by dshokouhi** [comment]
So one issue here is that F-Droid seems to be delayed in releases.  We do not push releases to them, someone else does it.  3.0.2 was 9 days late so users may get confused when there is a gap and our documentation shows more than they can do.

**Comment by qwow5** [comment]
@dshokouhi Did not think of that, maybe adding a message that releases are delayed will help?

File: docs/core/android-flavors.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ id: 'android-flavors'
...
 
-The `minimal` flavor of the app does not require Google Play Services and can be found on the [releases](https://github.com/home-assistant/android/releases) section on GitHub as an APK. This flavor of the app does not offer location tracking nor does it have notifications. The only sensors that will not be available are: [Activity](/core/sensors.md#activity-sensor) and [Geocoded](/core/sensors.md#geocoded-location-sensor).
+The `minimal` flavor of the app does not require Google Play Services and can be found on [F-Droid](https://f-droid.org/en/packages/io.homeassistant.companion.android.minimal) or the [releases](https://github.com/home-assistant/android/releases) section on GitHub as an APK. This flavor of the app does not offer location tracking nor does it have notifications. The only sensors that will not be available are: [Activity](/core/sensors.md#activity-sensor) and [Geocoded](/core/sensors.md#geocoded-location-sensor).
```
Comment (dshokouhi) [inline docs/core/android-flavors.md:8]: Let's put github releases link first since it will always be the most up to date.

**Review by dshokouhi — CHANGES_REQUESTED** [review]
Lets make sure to mention F-droid can be delayed and list github releases link first since it will always be up to date.

File: docs/core/android-flavors.md (hunk 1 above)
Comment (qwow5) [reply docs/core/android-flavors.md]: Just added message about this.

File: docs/core/android-flavors.md [hunk 2]
```diff
@@ -5,6 +5,6 @@ id: 'android-flavors'
...
 
-The `minimal` flavor of the app does not require Google Play Services and can be found on [F-Droid](https://f-droid.org/en/packages/io.homeassistant.companion.android.minimal) or the [releases](https://github.com/home-assistant/android/releases) section on GitHub as an APK. This flavor of the app does not offer location tracking nor does it have notifications. The only sensors that will not be available are: [Activity](/core/sensors.md#activity-sensor) and [Geocoded](/core/sensors.md#geocoded-location-sensor).
+The `minimal` flavor of the app does not require Google Play Services and can be found on the [releases](https://github.com/home-assistant/android/releases) section on GitHub as an APK. It can also be installed from [F-Droid](https://f-droid.org/en/packages/io.homeassistant.companion.android.minimal), however, since F-Droid builds new releases themselves, updates may be delayed. This flavor of the app does not offer location tracking nor does it have notifications. The only sensors that will not be available are: [Activity](/core/sensors.md#activity-sensor) and [Geocoded](/core/sensors.md#geocoded-location-sensor).
```
Comment (dshokouhi) [inline docs/core/android-flavors.md:8]: do they really build the release themselves or does someone grab the APK and submit it to their repo?

Also 3 commas feels a bit much for one sentence 😛

Comment (qwow5) [reply docs/core/android-flavors.md:8]: F-Droid builds the releases themselves. More information about their build process is on their website: `https://f-droid.org/en/docs/FAQ_-_General/#is-your-building-and-signing-process-secure`. Sorry about my bad grammar 🙁, I will fix that.

**Review by dshokouhi — APPROVED** [review]

**Pull request merged**